### ISSUE TITLE
MultilineRawStringIndentation false positive when using tab indentation

### ISSUE THREAD

**Issue by TWiStErRob** [detekt/detekt#5429]
## Expected Behavior
No flagging, full support of tabs as indentation on all styles of `"""` placement.

## Observed Behavior
Nicely formatted code is flagged:
```kotlin
	@Language("json")
	val body = """
		{
			"edi": 456,
			"cinema": 789,
			"date": ${fixtDate.toInstant().toEpochMilli()}
		}
	""".trimIndent()
	setBody(body)
```

## Steps to Reproduce
`MultilineRawStringIndentationSpec`:
```kotlin
        @Test
        fun `don't raise multiline raw strings if all have the correct indentation - tabs`() {
            val TAB = "\t"
            val code = """
                val a = 
                ${TAB}$TQ
                ${TAB}${TAB}Hello world!
                ${TAB}${TAB}How are you?
                ${TAB}$TQ.trimIndent()
            """.trimIndent()
            subject.compileAndLint(code)
            assertThat(subject.findings)
                .isEmpty()
        }
```
(note: I only used the `TAB` variable, to make sure it goes through GitHub properly.)

Although it might be safer even in Detekt codebase to use it to make it explicit, because this is hard to see:
![image](https://user-images.githubusercontent.com/2906988/196185898-b6827b65-bb8b-4c6f-b0c7-2286cff123b3.png)

## Context
NoTabs is not enabled by default:
https://github.com/detekt/detekt/blob/37ce2642eb5c1c2dcfef7a3fac4f2f9bf5e54e54/detekt-core/src/main/resources/default-detekt-config.yml#L607-L608
(and shouldn't be as it's a personal (project) style preference.)

## Your Environment

* Version of detekt used: 1.22.0-RC2
* Link to your project (if it's a public repository):

**Comment by TWiStErRob** [comment]
Here's another testcase:
```kotlin
        @Test
        fun `don't raise multiline raw strings if all have the correct indentation - tabs`() {
            val TAB = "\t"
            val code = """
                fun redirect(accessToken: String, refreshToken: String) {
                ${TAB}respond(
                ${TAB}${TAB}content = $TQ
                ${TAB}${TAB}${TAB}{
                ${TAB}${TAB}${TAB}    "access_token": "${'$'}{accessToken}",
                ${TAB}${TAB}${TAB}    "token_type": "fake_token_type",
                ${TAB}${TAB}${TAB}    "expires_in": 3600,
                ${TAB}${TAB}${TAB}    "refresh_token": "${'$'}{refreshToken}"
                ${TAB}${TAB}${TAB}}
                ${TAB}${TAB}$TQ.trimIndent(),
                ${TAB}${TAB}status = HttpStatusCode.FOUND
                ${TAB})   
                }
            """.trimIndent()
            subject.compileAndLint(code)
            assertThat(subject.findings)
                .isEmpty()
        }
`

**Comment by BraisGabin** [comment]
I thought about that while I was developing this but thought now one was using tabs anymore so I just didn't care 😂. I was wrong. And I need to link this video https://www.youtube.com/watch?v=SsoOG6ZeyUI

**Comment by TWiStErRob** [comment]
[Yeah... 😅](https://github.com/TWiStErRob/net.twisterrob.gradle/blob/cc11555e3ac274c8359aa2085a25a12b6e624923/config/detekt/detekt.yml#L73-L75)
```yaml
style:
  NoTabs:
    # I'm a tab guy, I don't pay for space in advance.
    active: false
```

Note that I would be happy even if it was not supported (i.e. converting this false positive to a false negative whenever tabs are used). No detection is better than "noise".
